examples/knative/powercli/kn-pcli-telegram/function.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: kn-pcli-telegram
+  labels:
+    app: veba-ui
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/maxScale: "1"
+        autoscaling.knative.dev/minScale: "1"
+    spec:
+      containers:
+        - image: projects.registry.vmware.com/veba/kn-pcli-telegram:1.0
+          envFrom:
+            - secretRef:
+                name: telegram-secret
+          env:
+            - name: FUNCTION_DEBUG
+              value: "false"
+---
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: veba-pcli-telegram-trigger
+  labels:
+    app: veba-ui
+spec:
+  broker: default
+  filter:
+    attributes:
+      type: com.vmware.event.router/event
```

Comment:
Feel free to drop

Reply:
Sorry, which line am I dropping? I see L30-33 highlighted and I thought we needed 30-32 to be able to filter on a specific VC Event?

Reply:
33 since 34 is specific enough IMHO